libraries/AP_Baro/AP_Baro_MS5611.h

```diff
@@ -37,7 +37,8 @@ class AP_Baro_MS56XX : public AP_Baro_Backend
         BARO_MS5611 = 0,
         BARO_MS5607 = 1,
         BARO_MS5637 = 2,
-        BARO_MS5837 = 3
+        BARO_MS5837_30BA = 3, // previously assumed variant
+        BARO_MS5837_02BA = 4 // Add new type here
```

Comment:
```suggestion
        BARO_MS5837 = 3,
        BARO_MS5837_30BA = 3, // previously assumed variant
        BARO_MS5837_02BA = 4
```
- The original (general) code is still used, before the variant is determined
- Code comments shouldn't discuss what is new/changing, they should just reflect what is in the code